controllers/clusterurlmonitor/clusterurlmonitor_supplement.go

```diff
@@ -26,16 +28,31 @@ type BlackBoxExporter interface {
 	GetBlackBoxExporterNamespace() string
 }
 
+type ResourceComparer interface {
```

Comment:
if this is the same in both.. what about moving it to a common place and not duplicating?

Reply:
I will merge stuff like this in my PR in rather a common util lib, yes there are quite some things in ClusterURLMonitor and RouteMonitor Reconcile that are common. So we can maybe leave it here for now. I will address it